modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -0,0 +1,82 @@
+@component-name = "OSB Site Initializer EVP"
+definition {
+
+	var baseURL = PropsUtil.get("portal.url");
+
+	property custom.properties = "feature.flag.LPS-169981=true";
+	property osgi.modules.includes = "osb-site-initializer-evp";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer EVP";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new EVP Site") {
+			EVPUtils.addEVPSite();
+		}
+
+		task ("Sign as Employee") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+		}
+
+		task ("Navigate to EVP Site") {
+			EVPUtils.gotoEVP();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
+		}
+	}
+
+	@description = "This is a test for LPS-172949. Make sure the Edit button can only be clicked when the user role is Financial User."
+	@priority = 5
+	test EmployeeAndManagerCanNotEditOrganization {
+		task ("Given the Employee user is in the Organization page") {
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("When the Employee user clicks on ellipsis button of an Organization with 'Awaiting Finance Approval' status") {
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization 2");
+		}
+
+		task ("Then the edit button dont appear") {
+			MenuItem.viewNotPresent(menuItem = "Edit");
+		}
+
+		task ("Given the Finance user is in the Organization page") {
```

Comment:
@juliosilveira77, should this task say "Manager" user instead of Finance user? 

The test description says that only the Finance user can edit. 
The task blocks are describing that the Finance user cannot edit.

```
Given the Finance user is in the Organization page
When the Finance user clicks on ellipsis button of an Organization with 'Awaiting Finance Approval' status
Then the edit button dont appear
```

Reply:
Sorry Nikki, it's like this because the scenario changed from Finance to Manager and I didn't change the task.